### DISCUSSION TITLE
Icon Request: fa-fleur-de-lis

### DISCUSSION THREAD

**Discussion by limbenjamin** [FortAwesome/Font-Awesome#21116] [Icon Requests]
The fleur-de-lis is a decorative symbol that has religious, political, dynastic, artistic, emblematic, and symbolic meaning.

It is widely used in many areas.

![](https://upload.wikimedia.org/wikipedia/commons/thumb/3/3c/Fleur-de-lis-fill.svg/200px-Fleur-de-lis-fill.svg.png)

**Comment by jlr7245** [comment]
+1

**Comment by gnowland** [comment]
Bump for this, Happy Mardi Gras!

Also, this SVG in the Wikimedia Commons would make for a _much_ nicer icon https://commons.wikimedia.org/wiki/File:Fleur-de-lis-fill.svg imho

**Comment by jmrossy** [comment]
+1

**Comment by elbel86** [comment]
+1

**Comment by klambacher** [comment]
+1

**Comment by RileyStarlight** [comment]
+1

**Comment by SoftwareBlair** [comment]
+1

**Comment by airtonj** [comment]
+1

**Comment by Mallfurion** [comment]
+1

**Comment by AuspiciousWeb** [comment]
+1

**Comment by kylejoe** [comment]
+1

**Comment by cbcoutinho** [comment]
+1

**Comment by trishah** [comment]
+1

**Comment by imanerd** [comment]
+1

**Comment by RobertLeePrice** [comment]
+1

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
⚜ [U+269C](https://emojipedia.org/emoji/%E2%9A%9C/) Fleur-de-lis

**Comment by g-berthiaume** [comment]
+1 ⚜️

**Comment by TheOldMan2000** [comment]
Great suggestion!

**Comment by simonperrier11** [comment]
+1 ! ⚜️

**Comment by tblobaum** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by jkgarner** [comment]
Gentlemen,

I have been waiting over 3 years for the addition of the fleur-de-lis icon and several new releases have been pushed without the addition of the icon. The initial request is now 10 years old. So I thought I would take matters in my own hand and see what I could gen up...

I downloaded the fleur-de-lis svg file from wikipedia and modified it to match the format of the svg files in the solid directory. I then added it to the directory of solid svg files as fleur-de-lis.svg.
I Ran my local testbed pointing to my local copy of fa. No go.

I then examined all the code in the fa directory and determined the only place that listed or mentioned icons was the js code.

I then looked at the js code.
In examining the icon javascript data, I determined that fields in the data were:
"icon-name": [width,height], [], "unique-hex-code", "SVG VECTOR DATA"],
I also determined that in my version of fa, the code f11f was not used.

To represent the icon, I wrote the line:
"fleur-de-lis": [350,359, [], "f11f", "<VECTOR_DATA_FROM_SVG_FILE>"],
Where <VECTOR_DATA_FROM_SVG_FILE> represents the actual vector data taken from the svg file.

I then added the line to solid.js, solid-min.js.
I Ran my local testbed pointing to my local copy of fa. No go.

I then added it to all.js and all-min.js and added the line (in 2 places):
I Ran my local testbed pointing to my local copy of fa.  It worked! YAY!

When I uploaded all  the altered files to my web server, both the fa files and my html using the icon, it showed the the error (circle ?) icon instead of showing the fleur-de-lis icon.

You can look at my web-page where I try to use it at http://k4jkg.com/
Next to the Scouts menu option, you will see the circle question (error?) icon instead of the fleur-de-lis icon

Right now I am trying to figure out why it works locally but not on the web.

Feel free to tell me why it does not work, or what I can change to make it work on the web.

Feel free to grab my code and drop it into FA as a free icon with your next update... there seems to be interest by many others.

I have added all.js as a text file to this message so that you can see the code...
Searching for "fleur" will take you to the 2 locations where I added it.
link: [all_js.txt](https://github.com/user-attachments/files/21681640/all_js.txt)

I have added the modified svg file as well 
link: [fleur-de-lis.svg](https://github.com/user-attachments/assets/1b2354c2-1e6f-45cf-b0ce-ae70fbc50074)

(displayed below)
![Image](https://github.com/user-attachments/assets/1b2354c2-1e6f-45cf-b0ce-ae70fbc50074)


Thank you,
Jim Garner
